src/bio/bio_buffer.c

```diff
@@ -975,12 +989,17 @@ int
 bio_iod_post(struct bio_desc *biod)
 {
 	struct bio_dma_buffer *bdb;
+	D_DEBUG(DB_IO,"biod->bd_buffer_prep %d", (int)biod->bd_buffer_prep);
```

Comment:
(style)  space required after that ',' (ctx:VxV)

Reply:
removed the debug message